PeerioTechnologies/peerio-desktop
# feat: create and delete image temp cache folder

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### PR author

#### Relevant issue tracker links  
https://peerio.tpondemand.com/entity/6851
#### Linked PRs in other repos
https://github.com/PeerioTechnologies/peerio-icebear/pull/82
#### Manual testing instructions  
This should do 3 things:
1. On login, creates a temp folder at `[OS_temp_directory]/peerio/[random_hex]`, and stores that location in TinyDB
2. On logout/exit/restart, deletes this temp folder
2. If, on login, temp folder already exists, deletes it and creates a new one
#### Additional notes
n/a

----
### Reviewer

* Localisation.
* Architectural decisions.
* Performance.
* Comments.
* Big feature tech design documentation.
* Readability.
* DRY.
* Can we extract something to SDK?
* Are there any bicycles invented that should be replaced with existing SDK
* Are there unit and e2e tests added?

----
### Repository owner
